wsmatrix@martin.zurowietz.de/ThumbnailsBoxOverride.js

```diff
@@ -163,7 +180,8 @@ var ThumbnailsBoxOverride = class {
       let spacing = themeNode.get_length('spacing');
 
       // Compute the scale we'll need once everything is updated
-      let totalSpacing = (this.getRows() - 1) * spacing;
+      let nWorkspaces = workspaceManager.n_workspaces;
+      let totalSpacing = (nWorkspaces - 1) * spacing;
```

Comment:
Why does this now use `n_workspaces` instead of `getRows`? This won't return the correct vertical spacing.